src/menubar.vala

```diff
@@ -187,24 +187,31 @@ public class MenuBar : Gtk.MenuBar
     }
 
     private List<Indicator.Object> indicator_objects;
+    private Gtk.MenuBar pMenubar;
 
     construct
     {
+        this.pMenubar = new Gtk.MenuBar ();
+        this.pMenubar.halign = Gtk.Align.END;
+        this.pMenubar.hexpand = true;
+        this.pMenubar.pack_direction = Gtk.PackDirection.RTL;
+        this.pMenubar.show ();
+        this.attach (this.pMenubar, 1, 0, 1, 1);
+        this.show ();
         add_style_class (this);
-
-        /* Handle high contrast background color */
         var menubar_style = new Gtk.CssProvider ();
 
         try
         {
-            menubar_style.load_from_data ("*.high_contrast { background-color: #ffffff; }", -1);
+            menubar_style.load_from_data ("* { background-color: transparent; } *.high_contrast { background-color: #ffffff; color: #000000; text-shadow: none; }", -1);
```

Comment:
ohhh... can we also make these colors configurable???

Reply:
Done. Now we can.